documents/management/weekly_reports/week_1.md

```diff
@@ -0,0 +1,45 @@
+| ----------- | --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------- | ------------------------------- | ------------------------------- | ------------------------------ |
+| Meaning     | The task didn't started yet | The task is at these beginnings and only the basis are made or are being made | The task is progressing well and the team enter in the heart of it | The end of the task approach and the team must continue at a good pace | This is the last part of the task, finish the task and pass it in review. | The task is entirely done and included. |
+
+## Ongoing Tasks
+
+|           Task          |      Priority        | Member Assigned                   | Progression | End Of Week objective  | Start Date |
+| :----------------------:| :------------------: | :-------------------------------: | :---------: | :--------------------: | :--------: |
+|     Project Charter     |        HIGH          |     Camille - Project Manager     |     20%     |          20%           | 05/17/2024 |
+|        Mock-ups         |        VERY HIGH     |     All team                      |     70%     |          100%          | 05/13/2024 |
+| Functional specification|        HIGH          |     Evan - Program Manager        |     10%     |          10%           | 05/17/2024 |
+
+## Finished Tasks
+
+|          Task              | Member Assigned                  | Start Date |  End Date  |
+| :---------------------:    | :------------------------------: | :--------: | :--------: |
+| Github Repository creation |    Mathias - Technical Lead      | 05/16/2024 | 05/16/2024 |
+
+## Team meeting
+
+- A meeting was made the 05/13/2024 at the beginning of the project to setup the project.
+- Another one was made the 05/16/2024 10 minutes before the end to keep track of the project.
+- Two else were made the 05/17/2024 at the beginning and the end of the project to share the progress and motivate the team.
+
+## Week feedback
+
+This week the project advanced a lot, The decision to all make a different mock-up and then sharing them together to keep the best ideas of everyone was made, then with these ideas, 2 groups were made to prepare two mock-ups to present to the client. But we need to be careful about the deadline for the Functional specifications.
+
+Globally, all the members of the team worked a lot and brought of their own in the advancement. They are motivated and feels great, some are confident with flutter and dart.
+
+## Next week planning
+
+The next week we will focus on the Functional specifications mainly.
+
+Next week concern's:
+- Finish entirely the Functional specifications
+- Finish the Project charter
+- Finish the Mock-ups
+- Gantt
```

Comment:
You wrote "Finish" for others line but not form Gantt and RACI what will you do with these 2?